scripts/ci/ci_pylint.sh

```diff
@@ -20,11 +20,29 @@
 # Script to run Pylint on all code. Should be started from root directory of Airflow:
 # ./scripts/ci/ci_pylint.sh
 
-set -ex
+set -x
 
 find . -name "*.py" \
 -not -path "./.eggs/*" \
 -not -path "./airflow/www/node_modules/*" \
 -not -path "./airflow/_vendor/*" \
 -not -path "./build/*" \
+-not -path "./tests/*" \
 -not -name 'webserver_config.py' | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --output-format=colorized
+RES_MAIN=$?
+
+find . -name "*.py" \
+-path "./tests/*" \
+-not -name 'webserver_config.py' | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --disable=missing-docstring --output-format=colorized
```

Comment:
While we're at it, let's also disable `no-self-use` for tests since Pylint complains about that in 99% of the tests.

Reply:
Also disable:

- `too-many-public-methods`
- `protected-access`

Reply:
Yeah. I am running it now on all tests and see whether I will find more similar errors that might make sense to be removed. But those two seems to be good candidates.

Reply:
There are a lot of invalid-names but I think those should be fixed for clarity of the tests @BasPH @mik-laj @ashb ?

Reply:
There are also "attribute-defined-outside-init" and "unused-argument" but I think those should be also fixed.

Reply:
invalid-names, attribute-defined-outside-init and unused-argument sound like valid errors to me, let's keep those enabled